test/evergreen.yml

```diff
@@ -2100,6 +2100,17 @@ tasks:
               exit 1
             fi
 
+  - name: compatibility-test-for-wiredtiger-releases
+    commands:
+      - func: "get project"
+      - command: shell.exec
```

Comment:
Could we use [subprocess.exe](https://github.com/evergreen-ci/evergreen/wiki/Evergreen-Best-Practices) here?

Reply:
Yes, subprocess.exec would make sense in this context to run a shell command.